src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const theadRows = document.querySelector('thead tr');
+const tbodyElement = document.querySelector('tbody');
+
+theadRows.addEventListener('click', e => {
+  const trElements = [];
+  let columnNumber;
+
+  for (let i = 0; i < theadRows.children.length; i++) {
+    if (theadRows.children.item(i) === e.target) {
+      columnNumber = i;
+    }
+  }
+
+  for (const trChild of tbodyElement.children) {
+    trElements.push(trChild);
+  }
+
+  trElements.sort((a, b) => {
+    let aData = a.children[columnNumber].textContent;
+    let bData = b.children[columnNumber].textContent;
+
+    if (columnNumber === 3) {
```

Comment:
better to use event.target.name, to understand which column was clicked and which data you should pick to apply sorting. Indices are not stable for this purpose. If I move `salary` column left, there will be `age` under columnNumber === 3, and your sorting wouldn't work properly